tests/integration/golang/aim/app/get_apps_test.go

```diff
@@ -0,0 +1,74 @@
+//go:build integration
+
+package run
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/G-Research/fasttrackml/pkg/database"
+	"github.com/G-Research/fasttrackml/tests/integration/golang/fixtures"
+	"github.com/G-Research/fasttrackml/tests/integration/golang/helpers"
+)
+
+type GetAppsTestSuite struct {
+	suite.Suite
+	client      *helpers.HttpClient
+	appFixtures *fixtures.AppFixtures
+}
+
+func TestGetAppsTestSuite(t *testing.T) {
+	suite.Run(t, new(GetAppsTestSuite))
+}
+
+func (s *GetAppsTestSuite) SetupTest() {
+	s.client = helpers.NewAimApiClient(helpers.GetServiceUri())
+
+	appFixtures, err := fixtures.NewAppFixtures(helpers.GetDatabaseUri())
+	assert.Nil(s.T(), err)
+	s.appFixtures = appFixtures
+}
+
+func (s *GetAppsTestSuite) Test_Ok() {
+	tests := []struct {
+		name             string
+		expectedAppCount int
+	}{
+		{
+			name:             "GetAppsWithExistingRows",
+			expectedAppCount: 2,
+		},
+		{
+			name:             "GetAppsWithNoRows",
+			expectedAppCount: 0,
+		},
+	}
+	for _, tt := range tests {
+		s.T().Run(tt.name, func(T *testing.T) {
+			defer func() {
+				assert.Nil(s.T(), s.appFixtures.UnloadFixtures())
+			}()
+
+			apps, err := s.appFixtures.CreateApps(context.Background(), tt.expectedAppCount)
+			assert.Nil(s.T(), err)
+
+			var resp []database.App
+			err = s.client.DoGetRequest(
+				"/apps",
+				&resp,
+			)
+			assert.Nil(s.T(), err)
+			for idx := 0; idx < tt.expectedAppCount; idx++ {
```

Comment:
sorry, but this test still look strange:
- Im still no sure about `GetAppsWithNoRows`. what is the reason to test this moment? Then we probably can test for example `GetAppsWithOneRow` or `GetAppsWithTwoRows`?
- you iterate through the expected count but inside the cycle compare always with `apps[0]`. it isn't really clear, why? So when you read this test it is unclear really.

Reply:
also this test is not fail as I can see only, because you in second case just use `expectedAppCount` which is equal `0` and cycle won't just work in that case, but data in any case will be in database.

Reply:
okay i'll take another look